content/docs/components/serving/kfserving.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "KFServing"
+description = "Using KFServing for serving Models"
+weight = 2
++++
+
+## KFServing with Kubeflow
+
+KFServing can be installed with Kubeflow v0.7, and KFServing Kustomize install files are [located in the manifests repo](https://github.com/kubeflow/manifests/tree/master/kfserving).
+
+KFServing provides a Kubernetes [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for serving ML Models on arbitrary frameworks. It aims to solve 80% of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and ONNX. 
```

Comment:
Change this:

```
ML Models
```

to this:
```
machine learning (ML) models
```

See earlier comments about capital letters, and also refer to the [style guide](https://www.kubeflow.org/docs/about/style-guide/#spell-out-abbreviations-and-acronyms-on-first-use) for information on using abbreviations.